packages/tooling/fast-tooling/src/data-utilities/web-component.ts

```diff
@@ -4,7 +4,7 @@ export interface WebComponentAttributeValues {
     /**
      * The value
      */
-    name: string;
+    name: string | number | boolean | null;
```

Comment:
What is the expected behavior for boolean and null values? I can understand an implicit conversion from number -> string but the other two aren't as clear to me. Do these *all* convert directly to strings? `attr="true"` and `attr="null"`?

Reply:
Good point, I'll take those out since they can be confusing.